src/FilmsList.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import FilmCard from './FilmCard';
+import { Film } from './api';
+
+type FilmsListProps = {
+  films: Film[];
+};
+
+export default function FilmsList(props: FilmsListProps) {
+  const { films } = props;
+  return (
```

Comment:
return всегда отделяй пустой строкой
```suggestion
  const { films } = props;
  
  return (
```